views/watch.jade

```diff
@@ -17,7 +17,11 @@ html(lang='en')
 		link(rel='stylesheet', href='/css/fonts.css')
 		link(rel='stylesheet', href='/css/sidebar.css')
 		link(rel='stylesheet', href='/css/watch.css')
-	
+		meta(name='mobile-web-app-capable', content='yes')
+		meta(name='apple-mobile-web-app-capable', content='yes')
+		meta(name='apple-mobile-web-app-status-bar-style', content='black')
+		meta(name='apple-mobile-web-app-title', content='#{event} Captions')
```

Comment:
These are for iOS. The first line enables this behavior, the second line tells iOS to render the OS toolbar of the phone (the clock, battery level, etc.) against a black background (which I think fits Aloft better than white), and the third line sets the default title of the bookmark that will be saved to the homescreen to ‘[event name] Captions’ (e.g., for https://aloft.nu/mkk/donutjs, it would be ‘donutjs Captions,’ though that title may of course be changed by the user).

Reply:
This is great! Thanks!